docs/B-Computations/types.md

```diff
@@ -15,7 +19,10 @@ After reading this section, you will be able to:
 
 A typed programming language uses a type system to interpret the bit streams in memory.  C is a typed programming language.  A type is the rule that defines how to store values in memory and which operations are admissible on those values.  A type defines the number of bytes available for storing values and hence the range of possible values.  We use different types to store different information.  The relation between types and raw memory is illustrated in the figure below.
```

Comment:
Hi, there! For the paragraph in here, it seems alright in a real page, but wouldn't that be better if we use single space ` ` 
 to break sentences rather than 2 spaces`  `?

Reply:
Hi Derek, I completely overlooked this. Thanks for pointing it out.